docs/tutorials/fundamentals/installation.md

```diff
@@ -223,6 +224,23 @@ For example, if you would like to install napari on python 3.10:
 pip install napari[all, pyqt] -c constraints_py3.10.txt
 ```
 
+### Problem with `numpy<2` on macOS arm
```

Comment:
@melissawm How to get a nice anchor for this, for easier link to this part of documentation?

Reply:
@Czaki 

```
(anchor-text)=
# Actual text
```

Ref: https://jupyterbook.org/en/stable/reference/cheatsheet.html#target-headers

Reply:
We also autogenerate anchors for sections until level 4 (https://github.com/napari/docs/blob/b979d31312a57575c5d0ee98f7463d2de80bac41/docs/conf.py#L197) and you can see the autogenerated anchors like so:

```
$ myst-anchors -l 3 docs/tutorials/fundamentals/installation.md
```

Because there are (to me) non intuitive rules around special characters since you have quotes and a `<` in the section title I would create a manual anchor like Juan said.